project.clj

```diff
@@ -13,21 +13,28 @@
   ;; requires lein 2.2.0+.
   :pedantic? :abort
 
+  :exclusions [org.clojure/clojure]
```

Comment:
If upstream projects are already depending on 1.7 and *not* excluding it, that seems to be what should be done here as well, for the sake of consistency.  I don't think that's a breaking change.